src/commands/appdistribution-distribute.ts

```diff
@@ -2,7 +2,7 @@ import * as fs from "fs-extra";
 
 import * as Command from "../command";
 import * as utils from "../utils";
-import * as requireAuth from "../requireAuth";
+import { requireAuth } from "../requireAuth";
```

Comment:
In your other TS refactor you export a default instead of exporting a named function. Is there a pattern you're following that I'm missing, or is it inconsistent? FWIW default exports kinda suck and I'd probably rather just move toward named exports for everything.

Reply:
I think default exports have their place, but I think the _worst_ thing is mixing named and default exports in the same file.

The command files, for example, are a little tricky to automate in their imports if they're named (because the names could be different), but we can do some other tricks to fix it as well.

There's more conversation around those that can be had, but yes, generally, I prefer named exports